### PR TITLE
convert icon-trash to fas fa-trash

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .

### Summary of Changes
changes the trash icon in actions dropdown to "fas fa-trash"

### Testing Instructions
apply pr.
inspect action dropdown "trash" icon.
verify icon class is "fas fa-trash"

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

### Documentation Changes Required
none

### CODE REVIEW

**Comment by N6REJ** [comment]
@sandewt

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on b2648c46a4130535b83c9997d5158162dc274c65<br /><br />Tested successfully in Beta4-dev of 24 August.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on b2648c46a4130535b83c9997d5158162dc274c65<br /><br />Joomla! 4.0.0-beta4-dev Development [ Mañana ] 29-July-2020 18:21 GMT

Tested links: see #30464
administrator/index.php?option=com_menus&view=items&menutype=0main-menu-blog
administrator/index.php?option=com_menus&view=items&menutype=mainmenu<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by hans2103** [comment]
I have tested this item :white_check_mark: successfully on 8028ea02dc7ad52a32811eb103f5031935e9880c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by hans2103** [comment]
@sandewt

**Comment by infograf768** [comment]
Restarted drone

**Comment by sandewt** [comment]
Sorry, but if we take a closer look, we show that not all spaces are placed correctly yet, for example:
I don't know if the intention is to change it now? Or in another PR?

- space is missing:
https://github.com/joomla/joomla-cms/blob/8028ea02dc7ad52a32811eb103f5031935e9880c/layouts/joomla/button/iconclass.php#L15

- two spaces:
https://github.com/joomla/joomla-cms/blob/8028ea02dc7ad52a32811eb103f5031935e9880c/layouts/joomla/toolbar/iconclass.php#L23

And so on..

[EDIT]

**Comment by hans2103** [comment]
@sandewt this can be fixed in https://github.com/joomla/joomla-cms/pull/30470 where I would like to merge two identical JLayout files into one.

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 8028ea02dc7ad52a32811eb103f5031935e9880c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by infograf768** [comment]
Restarted drone

File: layouts/joomla/toolbar/iconclass.php [hunk 1]
```diff
@@ -115,8 +120,7 @@
 {
 	$icon = 'fas fa-arrows-alt';
 }
-else
-{
+else{
```
Comment (sandewt) [inline layouts/joomla/toolbar/iconclass.php]: missing space, should be:  ```else {```

Comment (infograf768) [reply layouts/joomla/toolbar/iconclass.php]: I would rather go back to former format, i.e.
```
else
{
```

Comment (sandewt) [reply layouts/joomla/toolbar/iconclass.php]: In joomla 4, both variations appear in the code.
The manuel comes with the following if-else example:
[https://github.com/joomla/coding-standards/blob/master/manual/php.md](https://github.com/joomla/coding-standards/blob/master/manual/php.md)

**Comment by infograf768** [comment]
`system-test-mysql` still failing
```
 I trash article "Test Article"
   I am on page "/administrator/index.php?option=com_con..."
  [GET] http://localhost/test-install/administrator/index.php?option=com_content&view=articles
   I get config "timeout"
   I wait for element {"id":"filter_search"},90
   I get config "timeout"
   I wait for element {"id":"filter_search"},90
   I fill field {"id":"filter_search"},"Test Article"
   I click {"xpath":"//button[@aria-label='Search']"}
   I see "Test Article"
   I check all results 
  Selecting Checkall button
   I click toolbar button "Action"
   I wait 2
   I click toolbar button "trash"
  Screenshot and page source were saved into '/********/src/tests/Codeception/_output/' dir
 FAIL 
```

File: layouts/joomla/toolbar/iconclass.php (hunk 1 above)
Comment (infograf768) [reply layouts/joomla/toolbar/iconclass.php]: @sandewt 
I quote the manual https://github.com/joomla/coding-standards/blob/master/manual/php.md#an-if-else-example
```
else
{
	echo 'A white lie';
}
```

> In joomla 4, both variations appear in the code.

Alas yes, although the variation `else {` is in very few files. I guess they should be corrected.

Comment (N6REJ) [reply layouts/joomla/toolbar/iconclass.php]: @infograf768 wasn't supposed to have changed.. fixed it.

Comment (sandewt) [reply layouts/joomla/toolbar/iconclass.php]: >I quote the manual ..

Useful to know hwo to quote.

**Comment by N6REJ** [comment]
@richard67 @infograf768 drone passing now.

**Comment by infograf768** [comment]
Back to Pending. This PR has to be tested again.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on a2d4a663b172dbf82f2f3ae6b551aa4830c7b127<br /><br />Joomla! 4.0.0-beta4-dev Development [ Mañana ] 29-July-2020 18:21 GMT<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on a2d4a663b172dbf82f2f3ae6b551aa4830c7b127<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30468">issues.joomla.org/tracker/joomla-cms/30468</a>.</sub>

**Comment by N6REJ** [comment]
closing since we've merged it https://github.com/joomla/joomla-cms/pull/30470

**Pull request closed without merging**